docs/General/createCIRAConfig.md

```diff
@@ -1,6 +1,17 @@
-[Client Initiated Remote Access (CIRA)](../Glossary.md#c) enables a CIRA-capable edge device to initiate and establish a persistent connection to the MPS rather than the MPS initiating contact with the edge device.  This enables access to edge devices that might be harder to reach behind a firewall or a firewall process, such as network address translation (NAT).
+Out-of-band (OOB) manageability is a separate channel of remote management that allows administrators to securely access [Intel® Active Management Technology (Intel® AMT)](../Glossary.md#i) devices at a hardware level, beneath the operating system. This is accomplished with [Client Initiated Remote Access (CIRA)](../Glossary.md#c), which enables a CIRA-capable edge device to initiate and establish a persistent connection to the MPS.
 
-A CIRA Configuration provides the Remote Provisioning Client (RPC) the required information to establish the edge device's CIRA connection to the MPS Server during the activation process. 
+With this persistent connection, administrators can execute various system actions on the remotely managed device, including:
+
+* reboot
+* reset
+* power on and power up
+* power off and power down
+* boot to BIOS 
+* reset to BIOS
```

Comment:
Any particular reason this list isn't capitalized?

Reply:
Yeah, I was trying to get the user accustomed to the idea of these as information that can get passed or injected into the programmatic process. 

But I also don't have a strong feeling about this, so I changed it to capitalized.